packages/image_picker/tizen/src/image_resize.cc

```diff
@@ -0,0 +1,271 @@
+// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
+// Copyright 2019 The Flutter Authors. All rights reserved.
```

Comment:
Where did you get this file originally from?

Reply:
I referred to ImageResizer.java of Android port.

Reply:
Thanks!